config/vite-plugin-handlebars-loader.js

```diff
@@ -0,0 +1,34 @@
+// TODO: Export this to a vite plugin
```

Comment:
This is largely based off of the rollup plugin which would be vite compatible except that it uses `require` instead of `import`. We should open source this and extract it back for the community